challenge5/index.html

```diff
@@ -0,0 +1,292 @@
+          },
+          fontSize: {
+            button: ["18px", "60px"],
+          },
+        },
+      },
+    };
+  </script>
+</head>
+
+<body class="bg-background z-0 font-sans">
+  <header class="flex flex-row px-8 pt-2 pb-15 justify-between">
+    <div class="flex flex-row">
+      <img src="./assets/icons/logo.svg" alt="Logo" />
+    </div>
+    <div>
+      <div class="hidden md:flex">
+        <ul class="flex flex-row gap-x-5 pl-10">
+          <li class="font-medium">Home</li>
+          <li class="text-gray-400 font-medium">Find a doctor</li>
+          <li class="text-gray-400 font-medium">Apps</li>
+          <li class="text-gray-400 font-medium">Testimonials</li>
+          <li class="text-gray-400 font-medium">About us</li>
+        </ul>
+      </div>
+      <div class="flex md:hidden">
+        <img src="./assets/icons/List.svg" alt="Menu" />
+      </div>
+    </div>
+  </header>
+  <main class="flex flex-col overflow-hidden">
+    <section class="relative flex flex-col md:flex-row pl-9 pr-11.5 justify-center grow">
+      <div class="flex flex-col pr-10 pb-9 gap-y-6 self-center max-w-md">
+        <h1 class="pr-12 text-3xl font-bold">
+          Virtual healthcare for you
+        </h1>
+        <p class="text-lg font-light text-gray-400">
+          Trafalgar provides progressive, and affordable healthcare,
+          accessible on mobile and online for everyone
+        </p>
+        <button class="bg-secondary_color text-white rounded-full text-button px-10 w-button hover:opacity-80">
+          Consult today
+        </button>
+      </div>
+      <div class="flex flex-auto md:w-2/3 max-w-xl">
+        <img src="./assets/imgs/imgSection1.svg" alt="Illustration" class="flex flex-auto md:w-2/3" />
+      </div>
+    </section>
+
+    <section class="relative flex flex-col px-15 pt-20 self-center justify-center">
+      <img src="./assets/imgs/bgImg.svg" alt="Background Image" class="absolute left-0 transform -rotate-160 ">
+      
+      <div class="flex flex-col md:w-2/3 gap-y-6 self-center relative z-10">
+        <h2 class="text-3xl font-bold text-center">
+          Our services
+        </h2>
+        <div class="self-center hidden md:inline">
+          <img src="./assets/icons/Rectangle.svg" alt="Vector" />
+        </div>
+        <p class="text-lg text-center font-light text-gray-400">
+          We provide to you the best choices for you. Adjust it to your
+          health needs and make sure your undergo treatment with our highly
+          qualified doctors you can consult with us which type of service is
+          suitable for your health
+        </p>
+      </div>
+
+      <div class="flex flex-col flex-wrap justify-center md:flex-row px-7 pt-20 gap-y-5 md:gap-8 relative z-10">
+        <div class="flex flex-col bg-primary_color md:w-1/4 px-8 rounded-2xl">
+          <div class="self-center py-5">
+            <img src="./assets/imgs/Frame.svg" alt="frame" />
+          </div>
```

Comment:
Is this div with a single child needed? Maybe we can move the classes to the `img` (same for other similar cases)